angular/projects/common/src/app/utils/record-constants.ts

```diff
@@ -48,8 +48,7 @@ export class Picklists {
     Inspection: { displayName: 'Inspection', _schemaName: 'InspectionNRCED' },
     Order: { displayName: 'Order', _schemaName: 'OrderNRCED' },
     RestorativeJustice: { displayName: 'Restorative Justice', _schemaName: 'RestorativeJusticeNRCED' },
-    Ticket: { displayName: 'Ticket', _schemaName: 'TicketNRCED' },
-    Warning: { displayName: 'Warning', _schemaName: 'WarningNRCED' }
+    Ticket: { displayName: 'Ticket', _schemaName: 'TicketNRCED' }
```

Comment:
Are we supposed to remove Warning?  I didn't see it mentioned in the commit msg.

Reply:
Yeah, apparently there is some fuzziness at the business level around whether or not Warnings should be on NRCED, and possibly if they are allowed to be even (due to some law/rule or something).  Lena was fine with removing it, for now, to appease the business.